Composer/packages/lib/indexers/src/extractSchemaProperties.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { DialogInfo, JsonSchemaFile } from '@bfc/shared';
+
+import { getBaseName } from './utils/help';
+
+export const extractSchemaProperties = (dialog: DialogInfo, jsonSchemaFiles: JsonSchemaFile[]): string[] => {
+  if (dialog.content?.schema) {
+    const schema = jsonSchemaFiles.find(
+      (file) => file.id === getBaseName(dialog.content.schema) || file.id === dialog.content.schema
```

Comment:
missing a cast here

Reply:
The build caught that too :)  Fortunately, it helped me find another missing case to handle.